setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup
+
+description = """
+Contains two packages for processing ClinVar data for Open Targets submission.
+* `vep_mapping_pipeline` maps variants (CHROM:POS:REF:ALT) to their most severe functional consequence according to
+  Ensembl VEP, as well as their Ensembl gene ID and name.
+* `repeat_expansion_variants` parses ClinVar variant_summary file and extracts information about repeat expansion
+  variants."""
+
+setup(
+    name='vep-mapping-pipeline',
+    version='',
+    packages=['vep_mapping_pipeline', 'repeat_expansion_variants', 'repeat_expansion_variants.test'],
+    url='',
+    license='',
+    author='ktsukanov',
+    author_email='ktsukanov@ebi.ac.uk',
```

Comment:
I don't want to take authorship from you, but if anyone uses this pipelines, does it make sense to redirect them to eva-dev? I don't really know how this package setup works.

Reply:
I didn't indent to claim solo authorship :-) This file was originally automatically generated using my PyCharm config, this is where the values come from.

Per Tim's suggestion, changed the `author` to “Kirill Tsukanov, European Variation Archive, EMBL-EBI” and `author_email` to eva-dev@ebi.ac.uk. Fixed in https://github.com/EBIvariation/vep-mapping-pipeline/pull/4/commits/80436465c9097037810d32912069efac49e8268f.